src/Components/OpenStackObjectStorageSystem.php

```diff
@@ -2,24 +2,31 @@
 
 namespace DreamFactory\Core\Rackspace\Components;
 
-use DreamFactory\Core\Enums\HttpStatusCodes;
+use DateTimeImmutable;
+use InvalidArgumentException;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Config;
+
 use DreamFactory\Core\Exceptions\DfException;
 use DreamFactory\Core\Exceptions\BadRequestException;
 use DreamFactory\Core\Exceptions\NotFoundException;
+use DreamFactory\Core\Exceptions\NotImplementedException;
 use DreamFactory\Core\Exceptions\RestException;
 use DreamFactory\Core\File\Components\RemoteFileSystem;
 use DreamFactory\Core\Utility\Session;
 use DreamFactory\Core\Utility\FileUtilities;
-use Guzzle\Http\Exception\BadResponseException;
-use Guzzle\Http\Exception\ClientErrorResponseException;
-use InvalidArgumentException;
-use OpenCloud\Common\Exceptions\ObjFetchError;
-use OpenCloud\Rackspace;
-use OpenCloud\OpenStack;
-use OpenCloud\Common\Collection;
-use OpenCloud\ObjectStore\Service;
-use OpenCloud\ObjectStore\Resource\Container;
-use OpenCloud\ObjectStore\Resource\DataObject;
+use OpenStack\Common\Error\BadResponseError;
+use OpenStack\Common\Error\BaseError;
+use OpenStack\ObjectStore\v1\Service;
+use OpenStack\ObjectStore\Resource\Container;
+use OpenStack\ObjectStore\v1\Models\StorageObject;
```

Comment:
@anas-srikou these are different APIs. ObjectStore v1 for accessing blobs in cloud. Identity service v2 for request authentication. We also added support for Identity service v3, because it's the only option for recent OpenStack versions.